packages/aft/lib/src/commands/bootstrap_command.dart

```diff
@@ -70,6 +70,8 @@ const amplifyconfig = \'\'\'{
   "UserAgent": "aws-amplify-cli/2.0",
   "Version": "1.0"
 }\'\'\';
+
+const amplifyEnvironments = <String, String>{};
```

Comment:
Just playing with some ideas here. Integ tests would benefit from some ability to see all environments in a project - and we need a solution to this for our customers as well at some point.

Reply:
The purpose of adding it to the bootstrap command is so that analysis errors go away after bootstrapping, correct?

Reply:
Exactly, yeah. This is the format expected in auth integration tests now, so having the dummy config align with that allows analysis to pass.